src/test/java/de/tonypsilon/bmm/backend/division/service/DivisionServiceTest.java

```diff
@@ -63,33 +64,34 @@ void testGetAllDivisionsOfSeasonByLevel() {
         when(divisionRepository.findBySeasonId(1L)).thenReturn(List.of(
            landesliga, stadtligaA, stadtLigaB));
         SortedSetMultimap<Integer, DivisionData> actual = divisionService.getAllDivisionsOfSeasonByLevel(1L);
-        assertEquals(3, actual.size());
+        assertThat(actual.size()).isEqualTo(3);
 
-        assertEquals(1, actual.get(1).size());
+        assertThat(actual.get(1)).hasSize(1);
         Iterator<DivisionData> level1Actual = actual.get(1).iterator();
-        assertEquals(landesligaData, level1Actual.next());
+        assertThat(level1Actual.next()).isEqualTo(landesligaData);
         assertFalse(level1Actual.hasNext());
 
-        assertEquals(2, actual.get(2).size());
+        assertThat(actual.get(2)).hasSize(2);
         Iterator<DivisionData> level2Actual = actual.get(2).iterator();
-        assertEquals(stadtligaAData, level2Actual.next());
-        assertEquals(stadtLigaBData, level2Actual.next());
+        assertThat(level2Actual.next()).isEqualTo(stadtligaAData);
```

Comment:
Here it would be nice to use AssertJ's containsExactly method (which also checks the order of the elements).  See [here](https://www.javadoc.io/doc/org.assertj/assertj-core/latest/org/assertj/core/api/AbstractIterableAssert.html#containsExactly(ELEMENT...)) for Javadoc :)